common/auth/tlsConfigHelper.go

```diff
@@ -72,10 +75,15 @@ func NewTLSConfigWithCertsAndCAs(
 	clientAuth tls.ClientAuthType,
 	certificates []tls.Certificate,
 	clientCAs *x509.CertPool,
+	logger log.Logger,
 ) *tls.Config {
 	c := NewEmptyTLSConfig()
 	c.ClientAuth = clientAuth
 	c.Certificates = certificates
 	c.ClientCAs = clientCAs
+	c.VerifyConnection = func(state tls.ConnectionState) error {
+		logger.Info("successfully established incoming TLS connection", tag.HostID(state.ServerName))
```

Comment:
Do we think this should be Debug level log to avoid spamming if there are a lot of incoming connections? Or is Info fine?